internal/provider/lxd/provider.go

```diff
@@ -4,7 +4,6 @@
 package lxd
 
 import (
-	"context"
 	stdcontext "context"
```

Comment:
Nice spot!

I've been meaning to write a static-analysis tool to pick up this sort of thing. Perhaps I'll do so this week